coordinator/persistence-dse-next/src/main/java/io/stargate/db/cassandra/impl/StargateNodeView.java

```diff
@@ -0,0 +1,31 @@
+package io.stargate.db.cassandra.impl;
+
+import org.apache.cassandra.db.virtual.AbstractVirtualTable;
+import org.apache.cassandra.db.virtual.SimpleDataSet;
+import org.apache.cassandra.schema.TableMetadata;
+
+public abstract class StargateNodeView extends AbstractVirtualTable {
+  StargateNodeView(TableMetadata metadata) {
+    super(metadata);
+  }
+
+  DataSet completeRow(SimpleDataSet dataset, StargateNodeInfo info) {
```

Comment:
This has far more columns in Dse 6.8. Are those columns not required?

Reply:
Correct. I referenced the schema of the system tables as defined in the datastax/cassandra codebase. DSE has additional columns that are not present in datastax/cassandra (or apache/cassandra)